ci_config.json

```diff
@@ -34,17 +34,14 @@
   "abseil-cpp": {
     "_comment": ["Remove this once https://github.com/mesonbuild/meson/pull/9394 is released"],
```

Comment:
Comment here says it all.

Might as well get rid of these package entries completely.